### PR TITLE
Preparation for V2 release

### PR DESCRIPTION
**Pull request by atztogo**
- Add `changes_at_v2.md` -> renamed to `releases.md`.

### CODE REVIEW

**Comment by atztogo** [comment]
Almost done. Note `version.h` is set 2.0.0! I will merge this PR, then @lan496, could you have a look at the documentation? 

Especially, the `magnetic_symmetry_flags` page (simplified by commenting out the md). I added more info about `std_rotation_matrix` in the `dataset` page. I added the `releases` page. `spglib/ChangeLog` is no more necessary by this page and git-log. So I will stop writing changelog in `ChangeLog`. Now all doc files are md.

On the surface (i.e. for those functions that had been written in the old doc), I think this release is not backward incompatible, so the surprise is expected to be kept small.

File: doc/dataset.md [hunk 1]
```diff
@@ -194,13 +194,28 @@ idealization {math}`( \bar{\mathbf{a}}_\mathrm{s} \;
...
+\mathbf{c}_\mathrm{s} ).
 ```
 
+More precisely, this rotation matrix is an orthonormal matrix. Since {math}`(
+\mathbf{a}_\mathrm{s} \; \mathbf{b}_\mathrm{s} \; \mathbf{c}_\mathrm{s} )` can
+be distored, to make {math}`\boldsymbol{R}` orthonormal, it is calculated as
+follows. Make cubes of {math}`( \mathbf{a}_\mathrm{s} \; \mathbf{b}_\mathrm{s}
+\; \mathbf{c}_\mathrm{s} )` and {math}`( \bar{\mathbf{a}}_\mathrm{s} \;
+\bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )` by
+{math}`\mathrm{L} = \begin{pmatrix}\frac{\mathbf{a}}{|\mathbf{a}|} &
```
Comment (lan496) [inline doc/dataset.md:212]: $\mathbf{c} \times \mathbf{a}$ is replaced with $(\mathbf{a} \times \mathbf{b}) \times \mathbf{a}$ instead?
https://github.com/spglib/spglib/blob/5e98f3241e9d429f0f9168d7f30af4daa551058b/src/refinement.c#L1546-L1547

Comment (atztogo) [reply doc/dataset.md:212]: You are correct!

**Review by lan496 — COMMENTED** [review]
I skim through the document and leave a comment about possible typos.
The others look good!

**Comment by atztogo** [comment]
Thanks @lan496. The point of your correction has been fixed, https://spglib.github.io/spglib/develop/dataset.html. Now very close to the release v2.0.

**Pull request merged**